content/client-server-api/_index.md

```diff
@@ -2042,12 +2036,16 @@ able to accurately aggregate the events.
 #### Relationships API
 
 To retrieve the child events for a parent from the server, the client can call the
-following endpoint with relevant information. This endpoint does not aggregate the child
-events and is instead paginated: clients can perform local aggregation if needed. This
-allows clients to retrieve child events which do not require aggregation but still make
-use of `rel_type`.
+following endpoint. The events returned might require local aggregation by the client,
+however if those same events have bundles of their own then they should be present. A
```

Comment:
>  The events returned might require local aggregation by the client,

isn't this true of every endpoint that returns events?

Reply:
it is, and I'm convinced there's no value coming from the latter two thirds of the paragraph as well (per other comments).